self_hosted/ast2ir.oomph

```diff
@@ -660,18 +708,22 @@ class FunctionOrMethodConverter(
                 else:
                     # TODO: improve error for missing return value
                     # TODO: error for missing return statement
-                    self.code.push(new ir::Return(
-                        self.implicit_conversion(
-                            self.do_expression(ret.value.get()), self.returntype.get()
-                        ).get()
-                    ))
+                    self.code.push(new ir::Return(self.implicit_conversion(
+                        self.do_expression(ret.value.get()),
+                        self.returntype.get(),
+                        ast::locate_expression(ret.value.get())
```

Comment:
Is this third argument really needed? Or can location of first argument be used?